test/Router/IntegrationTest.php

```diff
@@ -458,16 +429,17 @@ public function testAllowedMethodsWhenNoHttpMethodsSet($adapter, $method)
         // This middleware is used just to check that request has successful RouteResult
         $middleware = $this->prophesize(MiddlewareInterface::class);
         $middleware->process(Argument::that(function (ServerRequestInterface $req) {
-            $routeResult = $req->getAttribute(RouteResult::class);
+                       $routeResult = $req->getAttribute(RouteResult::class);
 
-            Assert::assertInstanceOf(RouteResult::class, $routeResult);
-            Assert::assertTrue($routeResult->isSuccess());
-            Assert::assertFalse($routeResult->isMethodFailure());
+                       Assert::assertInstanceOf(RouteResult::class, $routeResult);
+                       Assert::assertTrue($routeResult->isSuccess());
+                       Assert::assertFalse($routeResult->isMethodFailure());
 
-            return true;
-        }), Argument::any())->will(function (array $args) {
-            return $args[1]->handle($args[0]);
-        });
+                       return true;
+                   }), Argument::any())
+                   ->will(function (array $args) {
+                       return $args[1]->handle($args[0]);
+                   });
```

Comment:
The spacing is wrong here. The original is good. there is no need to use so many spaces.

Reply:
The original is not clear, because we have `->will` starting in the middle of line.
When we have multiline chain call each part should be in new line, and now we have two options:
- align with `->` operator
- always just 4 spaces.

The sniff I wrote has this configuration and we can change it. Currently we are very inconsistent, because sometimes we have:
1. first call in the same line and every next in new line with one indent (4 spaces):
```php
$variable->call(...)
    ->method(...);
```

2. every call in new line with one indent (4 spaces):
```php
$variable
    ->call(....)
    ->method(....);
```

3. first call in the same line and next indented with `->` operator:
```php
$variable->call(...)
         ->method(...);
```

How the sniff is configured currently is to allow just 2 and 3 (case 1 is converted to 3).
We can reconfigure it to use 1 and 2 or make changes to have always 2.

Why I used 2 and 3? Because usually for short call 3 looks good and is readable, when we have something more complex then I would prefer to use 2.
Why I don't like 1? Because it's not aligned 🙄

Reply:
Spacing of the sample above looks fine to me; what did you object to, exactly, @xtreamwayz?

Reply:
The file is gone! No more evidence :)

I think the original had 4 space indentation and the new had 12 or even more spaces.